frontendUiVcsRootComboBoxImpl/build.gradle

```diff
@@ -0,0 +1,10 @@
+vavr()
```

Comment:
Hmm maybe this subproject could be a part of the existing `frontendUiRootImpl`? This seems to be too small & not distinctive enough to merit a separate subproject